src/redux/configureStore.js

```diff
@@ -0,0 +1,13 @@
+import { combineReducers, configureStore } from '@reduxjs/toolkit';
+import profileReducer from './profile/profile';
+import rocketsReducer from './rockets/rockets';
+import missionsReducer from './Missions/missions';
+
```

Comment:
- [ ] Please make sure to use redux logger in your application this is an example of how  to use it 
![image](https://user-images.githubusercontent.com/74744787/195981374-4da5ba2c-21f0-4b58-92cf-d29064ac5171.png)